components/Banner/examples/BannerTypicalUseExampleViewController.m

```diff
@@ -129,29 +130,73 @@ + (NSDictionary *)catalogMetadata {
   };
 }
 
+#pragma mark - UIViewController
+
+- (void)viewDidLayoutSubviews {
+  [super viewDidLayoutSubviews];
+
+  self.contentView.frame = self.view.bounds;
+  self.contentViewLabel.center = self.contentView.center;
+  CGRect exampleListTableViewFrame =
+      CGRectMake(0, self.view.bounds.size.height - exampleListTableViewHeight,
+                 self.view.bounds.size.width, exampleListTableViewHeight);
+  self.exampleListTableView.frame = exampleListTableViewFrame;
+
+  [self.bannerView sizeToFit];
+
+  // Adjust bannerViewContainer's frame
+  CGFloat topAreaInset = 0.0f;
+  if (@available(iOS 11.0, *)) {
+    topAreaInset = self.view.safeAreaInsets.top;
+  }
+
+  self.bannerView.frame = CGRectMake(0.0f, topAreaInset, self.bannerView.frame.size.width,
+                                     self.bannerView.frame.size.height);
+}
+
 #pragma mark - Internal helpers
 
 - (NSArray<BannerExampleUseInfo *> *)getBannerExampleList {
   NSMutableArray *bannerExampleList = [[NSMutableArray alloc] init];
   BannerExampleUseInfo *exampleUseInfo1 =
       [BannerExampleUseInfo infoWithIdentifier:@"example1"
-                                   displayName:@"Single Line"
+                                   displayName:@"Short Text with One Action"
                               exampleUseTarget:self
                             exampleUseSelector:@selector(showSingleLineStyleBanner)];
   [bannerExampleList addObject:exampleUseInfo1];
 
-  BannerExampleUseInfo *exampleUseInfo2 =
-      [BannerExampleUseInfo infoWithIdentifier:@"example2"
-                                   displayName:@"MultiLine Aligned Button"
+  BannerExampleUseInfo *exampleUseInfo4 =
+      [BannerExampleUseInfo infoWithIdentifier:@"example4"
+                                   displayName:@"Short Text with One Action (Icon)"
+                              exampleUseTarget:self
+                            exampleUseSelector:@selector(showSingleLineStyleBannerWithIcon)];
+  [bannerExampleList addObject:exampleUseInfo4];
+
+  BannerExampleUseInfo *exampleUseInfo5 =
+      [BannerExampleUseInfo infoWithIdentifier:@"example5"
+                                   displayName:@"Long Text with Two Short Actions"
                               exampleUseTarget:self
                             exampleUseSelector:@selector(showMultiLineAlignedButtonStyleBanner)];
+  [bannerExampleList addObject:exampleUseInfo5];
+
+  BannerExampleUseInfo *exampleUseInfo2 = [BannerExampleUseInfo
```

Comment:
Tip: Super-Nit: These variables are "out of order". Please consider rearranging them so they are numbered in-order.

Reply:
It was planned to be cleaned up, which I forgot to do. Thanks for bringing it. Addressed in 34a700237387d7c7f5cf43dbd42b855c18b38377.